components/tyk-mdcb/values.yaml

```diff
@@ -0,0 +1,280 @@
+    tag: v2.4.2
+    # image pull policy
+    pullPolicy: IfNotPresent
+
+  # image pull secrets to use when pulling images from repository
+  imagePullSecrets: []
+
+  # podAnnotations specifies annotations to be added in MDCB Pod
+  podAnnotations: {}
+  # podLabels specifie labels to be added in MDCB Pod
+  podLabels: {}
+
+  serviceAccount:
+    # Specifies whether a service account should be created
+    enabled: false
+    # Automatically mount a ServiceAccount's API credentials?
+    automount: true
+    # Annotations to add to the service account
+    annotations: {}
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
+
+
+  # securityContext holds pod-level security attributes for Tyk MDCB pod.
+  podSecurityContext: {}
+  # fsGroup: 2000
+
+  # containerSecurityContext holds container-level security attributes for Tyk MDCB container.
+  containerSecurityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+
+  service:
+    # type corresponds to the service type of Tyk MDCB Service.
+    type: ClusterIP
+    # The port at which the MDCB service can be found
+    port: 9090
+    # externalTrafficPolicy corresponds to the external traffic policy if the service type is load balancer
+    externalTrafficPolicy: Local
+    # annotations corresponds to the annotations which will be added into Tyk Dashboard Service.
+    annotations: {}
+
+  # ingress specifies Ingress rules for Tyk MDCB Service.
+  # Ingress is disabled by default.
+  ingress:
+    # enabled specifies whether ingress is enabled or not.
+    # Set it to true to enable Ingress for Tyk MDCB Service.
+    enabled: false
+    # className specifies your ingress controller class name below
+    className: ""
+    # annotations specifies annotations to be added on Ingress resource.
+    annotations: {}
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+    # hosts corresponds to the rules to be added on Ingress rules.
+    hosts:
+      - host: chart-example.local
+        paths:
+          - path: /
+            pathType: ImplementationSpecific
+    # tls corresponds to the tls configuration if Ingress rules use TLS
+    tls: []
+    #  - secretName: chart-example-tls
+    #    hosts:
+    #      - chart-example.local
+
+  resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+  # autoscaling configuration if kind IS NOT DaemonSet
```

Comment:
```suggestion
  # autoscaling configuration
```

Reply:
oh you mean since only supported kind is deployment, the comment is redundant. will update accordingly